tests/api_tests/nodes/test_views.py

```diff
@@ -2749,6 +2750,59 @@ def test_create_node_pointer_incorrect_type(self):
         assert_equal(res.status_code, 409)
         assert_equal(res.json['errors'][0]['detail'], 'Resource identifier does not match server endpoint.')
 
+def prepare_mock_wb_response(
+        node=None,
+        provider='github',
+        files=None,
+        folder=True,
+        path='/',
+        method=httpretty.GET,
+        status_code=200
+    ):
+    """Prepare a mock Waterbutler response with httpretty.
+
+    :param Node node: Target node.
+    :param str provider: Addon provider
+    :param list files: Optional list of files. You can specify partial data; missing values
+        will have defaults.
+    :param folder: True if mocking out a folder response, False if a file response.
+    :param path: Waterbutler path, passed to waterbutler_api_url_for.
+    :param str method: HTTP method.
+    :param int status_code: HTTP status.
+    """
+    node = node
+    files = files or []
+    wb_url = waterbutler_api_url_for(node._id, provider=provider, path=path, meta=True)
+
+    default_file = {
+        u'contentType': None,
```

Comment:
This is changing to a JSON API style response. I don't think it necessarily matters for what you're doing, but.